cloud_blobstore/__init__.py

```diff
@@ -117,7 +117,9 @@ def upload_file_handle(
             self,
             bucket: str,
             key: str,
-            src_file_handle: typing.BinaryIO):
+            src_file_handle: typing.BinaryIO,
+            content_type: typing.Optional[str],
```

Comment:
If these are optional, they need a default value.

Reply:
It's not actually required because `typing` will default them to `None`, but for readability I'll add default values.

Reply:
Are you sure?

```
In [7]: def ab(cd: typing.Optional[int]):
   ...:     print(cd)
   ...: 

In [8]: ab()
```

If I'm missing something and I'm wrong, we should avoid the needless `= None`.

Reply:
I believe I misinterpreted `Optional`. https://docs.python.org/3/library/typing.html#typing.Optional
I don't need to have the `typing.optional`, but I do need the `None`